### ISSUE TITLE
Clang C++20 Feature: P1972R0 - US105 Check satisfaction of constraints for non-templates when forming pointer to function

### ISSUE THREAD

**Issue by tstellar** [llvm/llvm-project#54008]
---
id: P1972R0
paper: https://wg21.link/p1972r0

**Comment by royjacobson** [comment]
Can we close this issue? @erichkeane 

AFAICT we've been checking for constraints there ever since clang 10: https://godbolt.org/z/9PEYescnW

**Comment by erichkeane** [comment]
Yep.  We can close this, but I need to update https://clang.llvm.org/cxx_status.html FIRST.  I'll do both.

**Comment by erichkeane** [comment]
Done here: https://github.com/llvm/llvm-project/commit/281b7eeb149d236dd4f2c1ab976cce361de4e057
